### PR TITLE
docs: split supported resources into sub-pages and add free resources

### PR DESCRIPTION
**Pull request by alikhajeh1**

### CODE REVIEW

File: blog/2020-08-17-infracost-cloud-costs-for-devs.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The complexity of cloud costs keeps increasing - When we were building PlanForCl
 
 We found existing tools fit in too late in the process and are not aimed at the people in control of the infrastructure. It's difficult to get cost estimations when you are building and deploying your services, which often leads to bill shock and no easy way to track down these costs. So we wanted to build a CLI tool that can plug into your existing development and operations processes and bring cost visibility to the engineers.
 
-Currently Infracost supports AWS and Terraform, but we will add support for more cloud vendors ([GCP](/docs/supported_resources#google-cloud-platform-gcp), [Azure](https://github.com/infracost/infracost/issues/64)) and other IaC tools ([Pulumi](https://github.com/infracost/infracost/issues/187)).
+Currently Infracost supports [AWS and Terraform](/docs/supported_resources/aws), but we will add support for more cloud vendors ([GCP](/docs/supported_resources/google), [Azure](/docs/supported_resources/azure)) and other IaC tools ([Pulumi](https://github.com/infracost/infracost/issues/187)).
```
Comment (vdmgolub) [inline blog/2020-08-17-infracost-cloud-costs-for-devs.md:21]: As we already have all these pages, decided to add a link for AWS and replace Azure's GH link to docs page.

File: docs/faq.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ title: FAQ
 
 Infracost has a [CLI](https://github.com/infracost/infracost) and a [Cloud Pricing API](https://github.com/infracost/cloud-pricing-api) backend service, as well as many [CI/CD integrations](/docs/integrations/cicd).
 
-The CLI parses the Terraform plan JSON file to find [supported resources](/docs/supported_resources) and uses [cost-related parameters](/docs/faq#example-request), such as the instance type or disk size, to find applicable cloud prices for that resource. The Cloud Pricing API [returns the prices](/docs/faq#example-response), which the CLI then uses to calculate the monthly costs. The results can be output in table, HTML or JSON format.
+The CLI parses the Terraform plan JSON file to find [supported resources](/docs/supported_resources/overview) and uses [cost-related parameters](/docs/faq#example-request), such as the instance type or disk size, to find applicable cloud prices for that resource. The Cloud Pricing API [returns the prices](/docs/faq#example-response), which the CLI then uses to calculate the monthly costs. The results can be output in table, HTML or JSON format.
```
Comment (vdmgolub) [inline docs/faq.md:10]: FYI l've also updated the links to overview section as `/supported_resources` is 404 now and Docusaurus doesn't perform the redirect. The redirect works well in the Footer though, so left it as is.

**Review by vdmgolub — COMMENTED** [review]
The changes look good. I've pushed 2 fixup commits with small fixes, please take a look. If all is good I'll rebase and merge 👍.

File: docs/faq.md (hunk 2 above)
Comment (alikhajeh1) [reply docs/faq.md:10]: Thanks, the redirects in docusaurus work with `npm run build` (not `npm start`)

**Pull request merged**